protocol/examples/messageData/coin/post_transaction_nooutput.json

```diff
@@ -0,0 +1,19 @@
+{
+    "object": "coin",
+    "action": "post_transaction",
+    "transaction_id": "47DEQpj8HBSa--TImW-5JCeuQeRkm5NMpJWZG3hSuFU=",
+    "transaction": {
+	    "version": 1,
+	    "inputs": [{
+		    "tx_out_hash": "47DEQpj8HBSa--TImW-5JCeuQeRkm5NMpJWZG3hSuFU=",
+		    "tx_out_index": 0,
+		    "script": {
+			    "type": "P2PKH",
+			    "pubkey": "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=",
+			    "sig": "CAFEBABE"
+		    }
+	    }],
+	    "outputs": [],
+	    "lock_time": 0
+    }
```

Comment:
I m confuse when exactly such case occur ? Because there is always an output, at least the same amount of coin back to the sender ?

Reply:
Of course, I agree that it should never happen. That's why if you take a look at the protocol test: https://github.com/dedis/popstellar/blob/ba118854ac6feedd51e67d6a66997fc7d84ad4f3/protocol/test/main.test.js#L191 
It is a test that should throw an error. Indeed in the example, you have some "wrong" test to check if it throws well an error.